examples/training/textual_inversion.py

```diff
@@ -257,10 +174,9 @@ def __getitem__(self, i):
             img = img[(h - crop) // 2 : (h + crop) // 2, (w - crop) // 2 : (w + crop) // 2]
 
         image = Image.fromarray(img)
-        if self.size is not None:
-            image = image.resize((self.size, self.size), resample=self.interpolation)
+        image = image.resize((self.size, self.size), resample=self.interpolation)
 
-        image = self.flip(image)
+        image = transforms.RandomHorizontalFlip(p=self.flip_p)
```

Comment:
I guess it's a bug here, image should be flipped instead of assigning to a transform